packages/jest-environment-jsdom/src/index.ts

```diff
@@ -38,6 +38,7 @@ export default class JSDOMEnvironment implements JestEnvironment<number> {
   private errorEventListener: ((event: Event & {error: Error}) => void) | null;
   moduleMocker: ModuleMocker | null;
   customExportConditions = ['browser'];
+  _configuredExportConditions?: Array<string>;
```

Comment:
```suggestion
  private _configuredExportConditions?: Array<string>;
```

Reply:
Oh nice, guess that's one step closer to using Node 14+ private properties :)